server/usage/sku/sku.go

```diff
@@ -0,0 +1,75 @@
+package sku
+
+// SKU is a unique, human-readable identifier tracking a specific usage count.
+// This is stored in the OLAP database as a low-cardinality string.
+// IMPORTANT: Do not cast arbitrary strings to this SKU type - use one of the
+// constants defined below.
+type SKU string
+
+// SKU constants - enumerated explicitly to ensure low cardinality.
+//
+// The format is roughly "<service>.<category>.<metric>". This hierarchical
+// convention allows easily querying subsets of usage data using prefix
+// matching, and when sorting by SKU, the usage data is naturally grouped by
+// service and type.
+const (
+	BuildEventsBESCount SKU = "build_events.build_event_stream.count"
+
+	RemoteCacheCASHits                   SKU = "remote_cache.content_addressable_storage.hits"
+	RemoteCacheCASDownloadedBytes        SKU = "remote_cache.content_addressable_storage.downloaded_bytes"
+	RemoteCacheACHits                    SKU = "remote_cache.action_cache_hits.hits"
+	RemoteCacheACCachedExecDurationNanos SKU = "remote_cache.action_cache.cached_execution_duration_nanos"
+
+	RemoteExecutionExecuteWorkerDurationNanos SKU = "remote_execution.execute.worker_duration_nanos"
+	RemoteExecutionExecuteWorkerCPUNanos      SKU = "remote_execution.execute.worker_cpu_nanos"
+	RemoteExecutionExecuteWorkerMemoryGBNanos SKU = "remote_execution.execute.worker_memory_gb_nanos"
```

Comment:
hahaha i think this can just stay as bytes?

Reply:
this is intentional since it's capturing peak memory usage multiplied by the time it was used (GB-nanos)